src/plugins/plugins.js

```diff
@@ -145,6 +145,7 @@ plugins.OpenInNewTabAction = OpenInNewTabAction;
 plugins.ReloadAction = ReloadAction;
 plugins.ClearData = ClearData;
 plugins.WebPage = WebPagePlugin;
+plugins.Darkmatter = DarkMatter;
```

Comment:
```suggestion
plugins.DarkMatterTheme = DarkMatter;
```

I'd like to expose this as `DarkMatterTheme` so it's clear what it is. We can't really go back and change the other two plugin names so it will be inconsistent, but it might communicate more effectively what the plugin does. What do you think?